core/trino-main/src/main/java/io/trino/sql/planner/ExpressionInterpreter.java

```diff
@@ -544,7 +544,7 @@ protected Object visitCoalesceExpression(CoalesceExpression node, Object context
                     .collect(toImmutableList()));
         }
 
-        private List<Object> processOperands(CoalesceExpression node, Object context)
+        private List<Object> processCoalesceOperands(CoalesceExpression node, Object context)
```

Comment:
How is the previous method not an accurate reflection of its purpose? It processes the operands of the given coalesce expression.

Reply:
it didn't have sufficiently meaningful name to me. dropping.